src/dom/dom.js

```diff
@@ -737,16 +743,18 @@ p5.prototype.createSelect = function() {
   };
 
   self.disable = function(value) {
-    if (value !== undefined && typeof value === 'string') {
+    if (value === undefined) {
+      this.elt.disabled = true;
+      return this;
+    } else if (typeof value === 'string') {
       for (let i = 0; i < this.elt.length; i++) {
-        if (this.elt[i].value === value) {
+        if (this.elt[i].value.toString() === value) {
           this.elt[i].disabled = true;
         }
       }
       return this;
-    } else if (arguments.length === 0) {
-      this.elt.disabled = true;
-      return this;
+    } else {
+      // TODO : Need a default behaviour here.
```

Comment:
What should be done here ?

Reply:
I think we should switch this to remove the undefined check, first check if it's a string, then put the lines 747, 748 under "else"

Reply:
Ok thanks.